lib/wallet/rpc.js

```diff
@@ -1,4 +1,4 @@
-/*!
+2015/*!
```

Comment:
Did you do `2015G` in vim and accidentally enter this lol

Reply:
yeah, probably

Reply:
Oh, no, it's actually that I tried to do 2015G in vscode because I was just in vim, instead of Ctrl+G 2015, which is correct